include/godot_cpp/variant/char_string.hpp

```diff
@@ -42,6 +42,7 @@ class CharString {
 	const char *_data = nullptr;
 	int _length = 0;
 
+	CharString(){};
```

Comment:
```suggestion
	CharString() {}
```
`clang-format` is a bit weird with this, but in theory the semicolon is indeed redundant when there's a code block.

Reply:
Fixed